docs/content/ingestion/schema-changes.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: doc_page
+---
+# Schema Changes
+
+Schemas for datasources can change at any time and Druid supports different schemas among segments.
+
+## Replacing Segments
+
+Druid uniquely 
+identifies segments using the datasource, interval, version, and partition number. The partition number is only present if 
```

Comment:
technically segments always have a partition number, even if it's just zero. We just don't print it as part of the identifier if it's zero.